### PR TITLE
Add a tooltip for slack notification

### LINKED ISSUE #18507: Add tooltip to remind webhook slack rate limit
How can we help you?

Add the tooltip to remind user that the slack has the rate limit when harbor sends webhooks to slack.  As described in the slack document(https://api.slack.com/docs/rate-limits) no more than 1 request per second.

### PR DESCRIPTION
**Pull request by AllForNothing**
1.Fixes #18507
2.Remind the users of Slack's rate limits
image

Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #(issue)

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Comment by AllForNothing** [comment]
@OrlinVasilev Can you help to review the English text for the added tooltip?
We want to remind the users of the rate limits for Slack(https://api.slack.com/docs/rate-limits)

File: src/portal/src/i18n/lang/en-us-lang.json [hunk 1]
```diff
@@ -431,7 +431,9 @@
         "EVENT_TYPE_REQUIRED": "Require at least one event type",
         "PAYLOAD_FORMAT": "Payload Format",
         "CLOUD_EVENT": "CloudEvents",
-        "PAYLOAD_DATA": "Payload Data"
+        "PAYLOAD_DATA": "Payload Data",
+        "SLACK_RATE_LIMIT": "Please be ware of the rate limits for Slack"
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/en-us-lang.json:435]:
```suggestion
        "SLACK_RATE_LIMIT": "Please be aware of Slack Rate Limits"
```

File: src/portal/src/i18n/lang/de-de-lang.json [hunk 2]
```diff
@@ -431,7 +431,8 @@
         "EVENT_TYPE_REQUIRED": "Mindestens ein Event Typ ist erforderlich",
         "PAYLOAD_FORMAT": "Payload Format",
         "CLOUD_EVENT": "CloudEvents",
-        "PAYLOAD_DATA": "Payload Data"
+        "PAYLOAD_DATA": "Payload Data",
+        "SLACK_RATE_LIMIT": "Please be ware of the rate limits for Slack"
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/de-de-lang.json:435]:
```suggestion
        "SLACK_RATE_LIMIT": "Please be aware of Slack Rate Limits"
```

File: src/portal/src/i18n/lang/es-es-lang.json [hunk 3]
```diff
@@ -432,7 +432,8 @@
         "EVENT_TYPE_REQUIRED": "Require at least one event type",
         "PAYLOAD_FORMAT": "Payload Format",
         "CLOUD_EVENT": "CloudEvents",
-        "PAYLOAD_DATA": "Payload Data"
+        "PAYLOAD_DATA": "Payload Data",
+        "SLACK_RATE_LIMIT": "Please be ware of the rate limits for Slack"
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/es-es-lang.json:436]:
```suggestion
        "SLACK_RATE_LIMIT": "Please be aware of Slack Rate Limits"
```

File: src/portal/src/i18n/lang/fr-fr-lang.json [hunk 4]
```diff
@@ -421,7 +421,8 @@
         "EVENT_TYPE_REQUIRED": "Au moins un type d'évènement est nécessaire",
         "PAYLOAD_FORMAT": "Payload Format",
         "CLOUD_EVENT": "CloudEvents",
-        "PAYLOAD_DATA": "Payload Data"
+        "PAYLOAD_DATA": "Payload Data",
+        "SLACK_RATE_LIMIT": "Please be ware of the rate limits for Slack"
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/fr-fr-lang.json:425]:
```suggestion
        "SLACK_RATE_LIMIT": "Please be aware of Slack Rate Limits"
```

File: src/portal/src/i18n/lang/pt-br-lang.json [hunk 5]
```diff
@@ -463,7 +463,8 @@
         "EVENT_TYPE_REQUIRED": "Pelo menos um tipo de evento é obrigatório",
         "PAYLOAD_FORMAT": "Payload Format",
         "CLOUD_EVENT": "CloudEvents",
-        "PAYLOAD_DATA": "Payload Data"
+        "PAYLOAD_DATA": "Payload Data",
+        "SLACK_RATE_LIMIT": "Please be ware of the rate limits for Slack"
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/pt-br-lang.json:467]:
```suggestion
        "SLACK_RATE_LIMIT": "Please be aware of Slack Rate Limits"
```

File: src/portal/src/i18n/lang/tr-tr-lang.json [hunk 6]
```diff
@@ -431,7 +431,8 @@
         "EVENT_TYPE_REQUIRED": "Require at least one event type",
         "PAYLOAD_FORMAT": "Payload Format",
         "CLOUD_EVENT": "CloudEvents",
-        "PAYLOAD_DATA": "Payload Data"
+        "PAYLOAD_DATA": "Payload Data",
+        "SLACK_RATE_LIMIT": "Please be ware of the rate limits for Slack"
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/tr-tr-lang.json:435]:
```suggestion
        "SLACK_RATE_LIMIT": "Please be aware of Slack Rate Limits"
```

File: src/portal/src/i18n/lang/zh-cn-lang.json [hunk 7]
```diff
@@ -430,7 +430,8 @@
         "EVENT_TYPE_REQUIRED": "请至少选择一种事件类型",
         "PAYLOAD_FORMAT": "载荷形式",
         "CLOUD_EVENT": "CloudEvents",
-        "PAYLOAD_DATA": "载荷数据"
+        "PAYLOAD_DATA": "载荷数据",
+        "SLACK_RATE_LIMIT": "请注意 Slack 的速率限制"
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/zh-cn-lang.json:434]:
```suggestion
        "SLACK_RATE_LIMIT": "Please be aware of Slack Rate Limits"
```

File: src/portal/src/i18n/lang/zh-tw-lang.json [hunk 8]
```diff
@@ -428,7 +428,8 @@
     "EVENT_TYPE_REQUIRED": "請至少選擇一種事件類型",
     "PAYLOAD_FORMAT": "Payload 格式",
     "CLOUD_EVENT": "CloudEvents",
-    "PAYLOAD_DATA": "Payload 資料"
+    "PAYLOAD_DATA": "Payload 資料",
+    "SLACK_RATE_LIMIT": "Please be ware of the rate limits for Slack"
```
Comment (OrlinVasilev) [inline src/portal/src/i18n/lang/zh-tw-lang.json:432]:
```suggestion
    "SLACK_RATE_LIMIT": "Please be aware of Slack Rate Limits"
```

**Review by OrlinVasilev — CHANGES_REQUESTED** [review]

**Comment by OrlinVasilev** [comment]
@AllForNothing suggested changes

**Comment by AllForNothing** [comment]
@OrlinVasilev Thanks for the comments. PR updated

**Review by OrlinVasilev — APPROVED** [review]
lgtm

**Review by chlins — APPROVED** [review]
lgtm

**Pull request merged**